Meta/Websites/man.serenityos.org/index.md

```diff
@@ -1,3 +1,17 @@
+![Help Icon](/icons/32x32/app-help.png)
+
+### Welcome to the SerenityOS on-line help system!
+
+This website is a browsable version of the SerenityOS manual pages.
```

Comment:
I hope the word _browsable_ is clear enough, but if it's not an alternative sentence could be:
"This website is a HTML rendering of..."
or
"...the SerenityOS manual pages rendered in HTML"